yarn-project/sequencer-client/src/publisher/l1-publisher.ts

```diff
@@ -961,14 +963,16 @@ export class L1Publisher {
 
   private async prepareProposeTx(encodedData: L1ProcessArgs) {
     const kzg = Blob.getViemKzgInstance();
+    const blobInput = Blob.getEthBlobEvaluationInputs(encodedData.blobs);
+    this.log.debug('validating blob input', { blobInput });
```

Comment:
```suggestion
    this.log.debug('Validating blob input', { blobInput });
```
I'm sorry, I can't help it